src/js/components/Data/DataForm.js

```diff
@@ -181,12 +181,18 @@ const resetPage = (nextFormValue, prevFormValue) => {
 };
 
 const transformTouched = (touched, value) => {
+  // DataFilters expects values for keys touched to evaluate to falsey to 
+  // not cause a badge. However any property value that is set back to its
+  // default/initial value that isn't undefined/null/false/0 will cause a
+  // badge this is particulary true for a 'range' which will always have
+  // a value.
+  //
+  // Should this instead determine touched by comparing against
+  // initial/default values?
+  //  
   const result = {};
   Object.keys(touched).forEach((key) => {
-    // special case _range fields
-    const parts = key.split('.');
-    if (parts[1] === formRangeKey) result[key] = value[parts[0]];
```

Comment:
Any rationale why _range was initially written with a special case but now isn't needed?

Reply:
The _xxxx property names are needed for the things in the forms that don't have an associated data property from the user like _search, etc. But the _range is used in a couple other spots to just treat the _range selector specially even though it has a property name from the data. Here it doesn't seem to be really needed since the flattened version of the property with _range on the end seems to be used as is later rather than strip off the _range suffix. Best I can tell...

Reply:
It looks like in `viewToFormValue` function there is this logic which would translate to the nested `_range` as opposed to flat:
```
Object.keys(result).forEach((key) => {
    if (
      typeof result[key]?.min === 'number' ||
      typeof result[key]?.max === 'number'
    ) {
      result[key] = { [formRangeKey]: [result[key].min, result[key].max] };
    }
  });
....
```

Will the changes cause any issues? Just trying to piece it all together in my head to understand where it's expecting `_range` to be a nested object inside of a property..

Reply:
Also investigating a bit myself to understand better.

Reply:
Thanks for checking!